server/portal/apps/workspace/api/views.py

```diff
@@ -542,4 +542,4 @@ def get(self, request):
             )
         )
 
-        return JsonResponse({"tabs": tabs, "definitions": definitions})
+        return JsonResponse({"tabs": tabs, "definitions": definitions})
```

Comment:
Looks like you dropped a newline here
```suggestion
        return JsonResponse({"tabs": tabs, "definitions": definitions})

```